src/rgw/rgw_cache.h

```diff
@@ -90,10 +90,11 @@ struct DataCache {
   std::map<string, ChunkDataInfo*> cache_map;
   std::list<string> outstanding_write_list;
   int index;
-  RWLock lock;
-  ceph::mutex cache_lock;
-  ceph::mutex req_lock;
-  ceph::mutex eviction_lock;
+  ceph::mutex lock = ceph::make_mutex("DataCache");
```

Comment:
can these be std::mutex and friend?  the only thing ceph wrappers can do is add overhead

Reply:
ack